### PR TITLE
Robert benavidez

### PR DESCRIPTION
**Pull request by robertbenavidez**

### CODE REVIEW

File: challenges/objects-arrays.js [hunk 1]
```diff
@@ -58,12 +95,22 @@ The resulting contact information should have a space between the first name and
...
 
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
 const uni = [];
+for (let i = 0; i < graduates.length; i++){
+  if(graduates[i].university.includes("Uni")){
+    uni.push(graduates[i].university)
+  }
+}
+
 console.log(uni);
```
Comment (angel-torres) [inline challenges/objects-arrays.js]: Make sure you complete all objectives

File: README.md [hunk 2]
```diff
@@ -30,14 +30,30 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. Describe the biggest difference between `.forEach` & `.map`.
 
+    The biggest difference between .forEach and .map is that .forEach mutates the original array and map does not. Map creates a new array and the change is represented there.
```
Comment (christie-roy) [inline README.md]: Good job picking up on this significant difference. It's good to know and will serve you going forward.

File: README.md [hunk 3]
```diff
@@ -30,14 +30,30 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. Describe the biggest difference between `.forEach` & `.map`.
 
+    The biggest difference between .forEach and .map is that .forEach mutates the original array and map does not. Map creates a new array and the change is represented there. 
+
 2. What is the difference between a function and a method?
 
+    The function operates over data through perameters, where a method operates over an object.
```
Comment (christie-roy) [inline README.md]: We talked about this in 1:1 but I'll drop the definition for you here too:
A method is a function housed inside of an object.

**Comment by christie-roy** [comment]
**MVP:**
_Higher Order Functions and Callbacks_
* [x] Review the contents of the [callbacks.js](assignments/callbacks.js) file.  Notice you are given an array at the top of the page.  Use that array to aid you with your functions.
* [x] Complete the problems provided to you but skip over stretch problems until you are complete with every other JS file first.

_Array Methods_
* [x] Review the contents of the [array-methods.js](assignments/array-methods.js) file.
* [x] Complete the problems provided to you but skip over stretch problems until you are complete with every other JS file first.
* [x] Notice the last three problems are up to you to create and solve.  This is an awesome opportunity for you to push your critical thinking about array methods, have fun with it.

_Closures_
* [x] Review the contents of the [closure.js](assignments/closure.js) file.
* [x] Complete the problems provided to you but skip over stretch problems until you are complete with every other JS file first.

**What Looks Good:**
- You accomplished MVP! Everything looks good.
- Nice use of the `for in` loop.

**Suggestions for Improvement:**
- Just practice stretch! I know you're already going to do that.
- Please be mindful of the git flow going forward. Make sure to branch, make a PR, and add me as a collaborator as soon as you start your project.

**Pull request merged**